scapy/contrib/mqtt.py

```diff
@@ -230,11 +231,32 @@ class MQTTSuback(Packet):
     ]
 
 
+class MQTTTopic(Packet):
+    name = "MQTT topic"
+    fields_desc = [
+        FieldLenField("len", None, length_of="topic"),
+        StrLenField("topic", "", length_from=lambda pkt:pkt.len)
```

Comment:
I'm surprised we do not have a PEP-8 test fail here. I though it should be:
```suggestion
        StrLenField("topic", "", length_from=lambda pkt: pkt.len)
```
That's a minor issue here, my comment is more about I though we had "something" that would catch it.